internal/workload/wrapper_test.go

```diff
@@ -79,7 +78,6 @@ var _ = Describe("Workload management", func() {
 			pod := &v1.Pod{ObjectMeta: metav1.ObjectMeta{Name: "pod1"}}
 
```

Comment:
`// then` comment is missing. To make the tests style consistent I would add it.

Reply:
It's on line 90. Or do I miss something?